site/learn/expert-advice/security/ssl-vs-tls.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: advice
+title: SSL vs. TLS
+description: In this blog, you’ll learn the difference between SSL and TLS, how they are used, and why you should use them.
+author: Akira Brand
+image: advice/steps-secure-auth-system/expert-advice-ten-steps-to-secure-your-authentication-system-header-image.png
+category: Security
+date: 2021-11-12
+dateModified: 2021-11-12
+---
+
+In this blog, you’ll learn the difference between SSL and TLS, when they are used, and why you should use them.
+
+*What are SSL and TLS?*
+
+SSL (secure socket layer) and TLS (transport layer security) are a set of protocols that are used when a computer is communicating with an external server to get information. The protocols ensure secure communication over the web by encrypting and securing any information sent over the world wide web.   
+
+*So what’s the difference between the two?*
+
+There isn’t a difference, actually. TLS replaced SSL, but since SSL has been around for longer, people still refer to TLS as SSL. Confused yet? Read on!
```

Comment:
I also am confused. You are saying they are the same, but one replaced the other? You might want to say "there isn't a functional difference" or something similar.